JudoPayDotNet/JudoPaymentsFactory.cs

```diff
@@ -138,10 +141,11 @@ private static bool PinPublicKey(object sender, X509Certificate certificate, X50
         /// <param name="judoEnvironment">Either the sandbox (development/testing) or live environments</param>
         /// <param name="token">Your API token (from our merchant dashboard)</param>
         /// <param name="secret">Your API secret (from our merchant dashboard)</param>
+        /// <param name="paymentSession">Your optional one-time payment session created with the same API token (replaces the API secret) </param>
         /// <returns>Initialized instance of the Judopay api client</returns>
-        public static JudoPayApi Create(JudoEnvironment judoEnvironment, string token, string secret)
+        public static JudoPayApi Create(JudoEnvironment judoEnvironment, string token, string secret, string paymentSession = null)
```

Comment:
An issue with extending the existing methods like this is that the api-version used by default (5.7) doesn't support Payment-Session authentication.     Rather than increasing the Api-Version used by the DotNet SDK for all methods (which might have unintended consequences), it may be simpler to leave these Create methods untouched and add another CreateWithPaymentSession method (that implicitly sets the api-version to 5.9 (or 5.11)), even if that method still return the same JudoPayApi object.    Another advantage of that approach would be that the secret parameter could be excluded from that CreateWithPaymentSession  method which simplifies it's use